presto-main/src/main/java/io/prestosql/sql/planner/DistributedExecutionPlanner.java

```diff
@@ -82,19 +86,32 @@
 
     private final SplitManager splitManager;
     private final Metadata metadata;
+    private final DynamicFilterService dynamicFilterService;
 
     @Inject
-    public DistributedExecutionPlanner(SplitManager splitManager, Metadata metadata)
+    public DistributedExecutionPlanner(SplitManager splitManager, Metadata metadata, DynamicFilterService dynamicFilterService)
     {
         this.splitManager = requireNonNull(splitManager, "splitManager is null");
         this.metadata = requireNonNull(metadata, "metadata is null");
+        this.dynamicFilterService = requireNonNull(dynamicFilterService, "dynamicFilterService is null");
     }
 
     public StageExecutionPlan plan(SubPlan root, Session session)
     {
         ImmutableList.Builder<SplitSource> allSplitSources = ImmutableList.builder();
         try {
-            return doPlan(root, session, allSplitSources);
+            // extract mapping of dynamic filter id to it's join node id
+            ImmutableMap.Builder<String, String> dynamicFilterSourcesBuilder = ImmutableMap.builder();
```

Comment:
make it `Map<String, PlanNodeId>` mapping

Reply:
done